src/components/favorites-card.tsx

```diff
@@ -0,0 +1,48 @@
+
+type FavoritesCardProps = {
+  isPremium: boolean;
+  imageCard: string;
+  price: number;
+  widthCard: string;
+  textCard: string;
+  typeCard: string;
```

Comment:
У нас и так пропсы для card, можно не дублировать 
```suggestion
  type: string;
```